.github/scripts/update-context-collection.py

```diff
@@ -0,0 +1,34 @@
+import strax
+import straxen
+from straxen.contexts import *
+from utilix import db
+import datetime
+
+
+# returns the list of dtype, hashes for a given strax context
+def get_hashes(st):
+    return set([(d, st.key_for('0', d).lineage_hash)
+                        for p in st._plugin_class_registry.values()
+                        for d in p.provides])
+
+
+def main():
+    for context in context_list:
+        # get these from straxen.contexts.*
+        st = eval("%s()" % context)
+        hashes = get_hashes(st)
+        hash_dict = {dtype: h for dtype, h in hashes}
+
+        doc = dict(name=context,
+                   date_added=datetime.datetime.utcnow(),
```

Comment:
Wont be `.utcnow()` different depending on your machines location? Will this be an issue?

Reply:
utcnow does not depend on the machine location -- it always returns the current time in UTC. I just tested comparing a machine in San Diego and a machine in Chicago